README.md

```diff
@@ -30,14 +30,38 @@ Full documentation [is available here](http://docs.docker.com/swarm/).
 You can download and install from source instead of using the Docker
 image. Ensure you have golang, godep and the git client installed.
 
+Note: The current developing version (0.4) requires golang 1.4.2 version
```

Comment:
:+1:

On Thu, Jul 30, 2015 at 4:13 PM, Andrea Luzzardi <notifications@github.com>
wrote:

> In README.md
> <https://github.com/docker/swarm/pull/1107#discussion_r35932258>:
>
> > @@ -30,14 +30,38 @@ Full documentation [is available here](http://docs.docker.com/swarm/).
> >  You can download and install from source instead of using the Docker
> >  image. Ensure you have golang, godep and the git client installed.
> >
> > +Note: The current developing version (0.4) requires golang 1.4.2 version
>
> Maybe "Starting from Swarm 0.4 golang 1.4.x is required" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/swarm/pull/1107/files#r35932258>.
>



-- 
Victor Vieux
Docker Core Engineer
vieux@docker.com